test/controllers/ideas_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class IdeasControllerTest < ActionController::TestCase
+  setup do
+    @idea = ideas(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:ideas)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create idea" do
+    assert_difference('Idea.count') do
+      post :create, idea: { description: @idea.description, title: @idea.title }
+    end
+
+    assert_redirected_to idea_path(assigns(:idea))
+  end
+
+  test "should show idea" do
+    get :show, id: @idea
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get :edit, id: @idea
+    assert_response :success
+  end
+
+  test "should update idea" do
+    patch :update, id: @idea, idea: { description: @idea.description, title: @idea.title }
+    assert_redirected_to idea_path(assigns(:idea))
+  end
+
+  test "should destroy idea" do
+    assert_difference('Idea.count', -1) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.